demo/Nonlinear Smoothing Test.ipynb

```diff
@@ -0,0 +1,914 @@
+{
```

Comment:
<p>I assume this notebook will be removed, since it is called <code>... Test</code>, otherwise please add text to explain what is happening  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>